commands

```diff
@@ -52,7 +52,8 @@ case "$1" in
     IMAGE=$(docker commit $ID)
     docker tag $IMAGE $DB_IMAGE
     # Launch container
-    ID=$(docker run -v $VOLUME -p 5432 -d $DB_IMAGE /usr/bin/start_pgsql.sh $DB_PASSWORD)
+    GATEWAY_IP=$(ip -f inet -o addr show docker0|cut -d\  -f 7 | cut -d/ -f 1)
```

Comment:
I think it would be better to use `docker inspect` to get the gateway IP address. For those of us with a custom bridge (in my case, I have `docker1`) - this will break for us.